fearless/Modules/AccountConfirm/BaseAccountConfirmInteractor.swift

```diff
@@ -13,13 +13,13 @@ class BaseAccountConfirmInteractor {
     let operationManager: OperationManagerProtocol
 
     init(
-        flow: AccountConfirmFlow,
+        flow: AccountConfirmFlow?,
         accountOperationFactory: MetaAccountOperationFactoryProtocol,
         accountRepository: AnyDataProviderRepository<MetaAccountModel>,
         operationManager: OperationManagerProtocol
     ) {
         self.flow = flow
-        shuffledWords = flow.mnemonic.allWords().shuffled()
+        shuffledWords = flow?.mnemonic.allWords().shuffled() ?? []
```

Comment:
if there are no words, this screen shoouldn't be presenter. you need to make this throwable maybe